_includes/free-software-logos/rockylinux.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="4096" height="4096" viewBox="0 0 192 192" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This actually renders on the website at 4096x4096, which is about 100 times the size of the rest of the logos.  It also displays in green while the rest of them are monochrome.  Is there a version of this logo that includes the name with it? The other logos in this section all include names with them.

Reply:
lmao I didn't see that, fixing that now

Reply:
And fixed. On a related note, [Boxy SVG](https://boxy-svg.com/) is wonderful.